src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/HealthCertifiedPersonViewModel.swift

```diff
@@ -222,15 +222,14 @@ final class HealthCertifiedPersonViewModel {
 
 	private var dateOfBirth: String? {
 		guard
-			let dateOfBirthString = healthCertifiedPerson.dateOfBirth,
-			let dateOfBirthDate = ISO8601DateFormatter.justLocalDateFormatter.date(from: dateOfBirthString)
+			let dateOfBirthString = healthCertifiedPerson.dateOfBirth
 		else {
 			return nil
 		}
 
 		return String(
 			format: AppStrings.HealthCertificate.Person.dateOfBirth,
-			DateFormatter.localizedString(from: dateOfBirthDate, dateStyle: .medium, timeStyle: .none)
+			DCCDateStringFormatter.formatedString(from: dateOfBirthString)
```

Comment:
If I can trust Figma, this one should stay in a localized format. I think the new formatting rules only apply to the `HealthCertificateViewModel`

Reply:
Then we need a way to parse and localize something like "1986" or "1986-11".
I'll create a task for that.